core/block_svg.ts

```diff
@@ -506,13 +516,8 @@ export class BlockSvg extends Block implements IASTNodeLocationSvg,
       this.workspace.getBlockDragSurface()!.translateSurface(
           newLoc.x, newLoc.y);
     } else {
-      (this.svgGroup_ as AnyDuringMigration).translate_ =
-          'translate(' + newLoc.x + ',' + newLoc.y + ')';
-      (this.svgGroup_ as AnyDuringMigration)
-          .setAttribute(
-              'transform',
-              (this.svgGroup_ as AnyDuringMigration).translate_ +
-                  (this.svgGroup_ as AnyDuringMigration).skew_);
```

Comment:
Why did you remove setting the skew value here? Won't this mess up the animation where the block wobbles when you disconnect it?

Or was the assignment in `setDragging` overwriting the skew value anyway, so it doesn't matter?

Reply:
AFAICT no, because the disconnect animation calculates skew (and updates `transform` using it) in a 10ms loop, so it will continue to do so and override the lack of skew from here. Experimentally there was no visually perceptible difference and the animation was still shown, but please do double check me on this!

Reply:
Pulled it down and checked, and I agree looks fine =) Sweetness!